lib/rubocop/cop/bundler/ordered_gems.rb

```diff
@@ -17,7 +17,8 @@ module Bundler
       #
       #   gem 'rspec'
       class OrderedGems < Cop
-        MSG = 'Gem `%s` should appear before `%s` in their gem group.'.freeze
+        MSG = 'Alphabetize gems: Gem `%s` should appear before `%s` '\
```

Comment:
Frankly I don't like this message that much. I'd add something like `(gems should be sorted in an alphabetical order within their section of the Gemfile)` or something like this.